doc/source/whatsnew/v1.6.0.rst

```diff
@@ -215,6 +214,7 @@ Indexing
 - Bug in :meth:`DataFrame.reindex` filling with wrong values when indexing columns and index for ``uint`` dtypes (:issue:`48184`)
 - Bug in :meth:`DataFrame.reindex` casting dtype to ``object`` when :class:`DataFrame` has single extension array column when re-indexing ``columns`` and ``index`` (:issue:`48190`)
 - Bug in :func:`~DataFrame.describe` when formatting percentiles in the resulting index showed more decimals than needed (:issue:`46362`)
+- :meth:`DataFrameGroupBy.fillna` and :meth:`SeriesGroupBy.fillna` now allow only keyword arguments to be passed except value(:issue:`48395`)
```

Comment:
space after value, and please use ``value``

Also please move to deprecations

Reply:
I received contradicting review about moving/not moving to deprecations in the previous comments.

I am kinda confused now regarding which one should I actually do?

Reply:
We have a section ``Other Deprecations`` where we announce all deprecations before changing behavior in a later release

Reply:
Hi, I could not find a section with the name `Other Deprecations` so moving it to deprecations for now.